src/m365/spo/commands/page/page-control-set.ts

```diff
@@ -31,10 +32,24 @@ class SpoPageControlSetCommand extends SpoCommand {
   }
 
   public commandAction(logger: Logger, args: CommandArgs, cb: (err?: any) => void): void {
-    Page
-      .getPage(args.options.name, args.options.webUrl, logger, this.debug, this.verbose)
-      .then((clientSidePage: ClientSidePage): Promise<ClientSidePageProperties> => {
-        const control: ClientSidePart | null = clientSidePage.findControlById(args.options.id);
+    const requestOptions: any = {
+      url: `${args.options.webUrl}/_api/SitePages/Pages/GetByUrl('sitepages/${encodeURIComponent(args.options.name)}')`,
+      headers: {
+        'content-type': 'application/json;charset=utf-8',
```

Comment:
Since we're not sending anything, I don't think we need to specify the `content-type`, do we?